conditional/blueprints/intro_evals.py

```diff
@@ -32,10 +32,24 @@ def display_intro_evals(internal=False, user_dict=None):
     log.info('Display Intro Evals Listing')
 
     # get user data
-    def get_fid_cm_count(member_id):
-        return len([a for a in FreshmanCommitteeAttendance.query.filter(
-            FreshmanCommitteeAttendance.fid == member_id)
-            if CommitteeMeeting.query.filter(CommitteeMeeting.id == a.meeting_id).first().approved])
+    def get_fid_cm(member):
```

Comment:
nit: plural

Reply:
?

Reply:
it's getting committee meetings, plural, right? Could the function name represent that?